### DISCUSSION TITLE
Getting PeakMatrix with getMatrixFromProject()

### DISCUSSION THREAD

**Discussion by ahyeonhwang** [GreenleafLab/ArchR#806] [Questions / Documentation]
Hello,

I am trying to access my PeakMatrix using getMatrixFromProject(proj,useMatrix = "PeakMatrix"). The dimensions of PeakMatrix are around 500,000 peaks by 350,000 cells. How much memory and number of cores are required to load the matrix? Do you have any suggestions for loading this? Also, what is the idx column used for in the rowData? 

Thank you for your consideration and time!

**Comment by rcorces** [comment]
I dont have a good estimate for memory used when loading that matrix - sorry.
The `idx` column is a unique index number given to each peak which is used in other functions to identify the peak. For example `idxATAC` here:
https://www.archrproject.com/bookdown/peak2genelinkage-with-archr.html
